libc/startup/baremetal/aie2/ldfile

```diff
@@ -0,0 +1,37 @@
+/*
+ * This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+ * See https://llvm.org/LICENSE.txt for license information.
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ * (c) Copyright 2024 Advanced Micro Devices, Inc. or its affiliates
+ */
+
+MEMORY
+{
+   program (RX) : ORIGIN = 0, LENGTH = 0x0020000
+   data (!RX) : ORIGIN = 0x20000, LENGTH = 0x0020000
+}
+ENTRY(_main_init)
```

Comment:
CHECK : shouldn't the entry be `__start` ?

Reply:
Well spotted! Yes, dead wrong, this.

Reply:
Note that this needs to be a published standard, because other tools need to generate ldfiles.   So you get to change it.... once.  Then document it and set it in stone.

Reply:
I'm not afraid to document __start as entry point, which will be recognizable by unix system programmers. It is actually only documenting crt0.o. If you write your own ldfile, you can also choose to supply your own startup code and place it at address 0x0. And even 0x0 is only necessary if you want to start executing at zero from hardware reset, which would require ROM PM.
I will add some comments here as documentation.